src/client/components/Profile/ProfileDetails.js

```diff
@@ -165,5 +176,4 @@ ProfileDetails.defaultProps = {
     'https://cdn.pixabay.com/photo/2016/08/31/11/54/user-1633249_960_720.png',
   phoneNumber: '+45---',
```

Comment:
You should change `defaultProps` to use `''` instead of these. If you need to explain to user what they need to enter, use the `placeholder` property on input.

Reply:
Implemented the above changes.